OpenImis.ModulesV2/InsureeModule/Repositories/FamilyRepository.cs

```diff
@@ -0,0 +1,282 @@
+                                    where (I.Chfid == chfid
+                                        && D.DistrictId == item
+                                        && F.ValidityTo == null
+                                        && I.ValidityTo == null
+                                        && V.ValidityTo == null
+                                        && W.ValidityTo == null
+                                        && D.ValidityTo == null)
+                                    select F.FamilyId)
+                                   .FirstOrDefault();
+
+                    response = imisContext.TblInsuree
+                                        .Where(i => i.ValidityTo == null && i.FamilyId == familyId)
+                                        .Join(imisContext.TblFamilies, i => i.FamilyId, f => f.FamilyId, (i, f) => f)
+                                        .Where(f => f.ValidityTo == null)
+                                        .Include(f => f.TblInsuree)
+                                            .ThenInclude(f => f.Photo)
+                                        .Select(f => FamilyModel.FromTblFamilies(f))
+                                        .FirstOrDefault();
+                }
+
+                if (response == null)
+                {
+                    return null;
+                }
+
+                return response;
+            }
+            catch (SqlException e)
+            {
+                throw e;
+            }
+            catch (Exception e)
+            {
+                throw e;
+            }
+        }
+
+        public int Create(EnrollFamilyModel model, int userId, int officerId)
+        {
+            try
+            {
+                string webRootPath = _hostingEnvironment.WebRootPath;
+
+                var enrollFamily = (Enrolment)model;
+
+                enrollFamily.FileInfo.UserId = userId;
+                enrollFamily.FileInfo.OfficerId = officerId;
+
+                var XML = enrollFamily.XMLSerialize();
+                var JSON = JsonConvert.SerializeObject(enrollFamily);
+
+                var EnrollmentDir = _configuration["AppSettings:Enrollment_Phone"];
+                var JsonDebugFolder = _configuration["AppSettings:JsonDebugFolder"];
+                var UpdatedFolder = _configuration["AppSettings:UpdatedFolder"];
+                var SubmittedFolder = _configuration["AppSettings:SubmittedFolder"];
+
+                var hof = "";
+
+                if (enrollFamily.Insurees.Any(x => x.isHead == "true" || x.isHead == "1"))
+                {
+                    hof = enrollFamily.Insurees.Where(x => x.isHead == "true" || x.isHead == "1").Select(z => z.CHFID).FirstOrDefault();
+                }
+                else hof = "Unknown";
+
+                var FileName = string.Format("{0}__{1}_{2}.XML.xml", hof, officerId.ToString(), DateTime.Now.ToString("dd-MM-yyyy HH-mm-ss.XML"));
```

Comment:
Why we have two times the extension xml: ".XML.xml"?

Reply:
There was such an entry in the comment but it is already corrected to `{0}__{1}_{2}.xml`